src/acceleration/BaseQNAcceleration.cpp

```diff
@@ -33,15 +33,15 @@ BaseQNAcceleration::BaseQNAcceleration(
     double                  initialRelaxation,
     bool                    forceInitialRelaxation,
     int                     maxIterationsUsed,
-    int                     timestepsReused,
+    int                     pastTimeWindowsReused,
```

Comment:
```suggestion
    int                     timeWindowsReused,
```
Isn't the "past" redundant as already covered by "reused"?

Reply:
I agree. Will change this.

Reply:
Done. https://github.com/precice/precice/pull/1001